### ISSUE TITLE
problem on running notebook：dif. result in the  "surprising perturbation..."

### ISSUE THREAD

**Issue by Siki-cloud** [tongshuangwu/polyjuice#13]
when i ran test code in  " Select surprising perturbations as counterfactual explanations" , the true label is 1,but why i got 0.  i dont know why?

  orig = (
      "How can I help a friend experiencing serious depression?",
      "How do I help a friend who is in depression?"
  )
  orig_label = 1

   # get a model
from transformers import pipeline, AutoTokenizer, AutoModelForSequenceClassification
import torch
model_name = "textattack/bert-base-uncased-QQP"
tokenizer = AutoTokenizer.from_pretrained(model_name)
model = AutoModelForSequenceClassification.from_pretrained(model_name)
# sentiment analysis is a general name in Huggingface to load the pipeline for text classification tasks.
# set device=-1 if you don't have a gpu
pipe = pipeline(
    "sentiment-analysis", model=model, tokenizer=tokenizer, 
    framework="pt", device=0 if is_cuda else -1, return_all_scores=True,padding=True,truncation=True)

# some wrapper for prediction
import numpy as np
def extract_predict_label(raw_pred):
    raw_pred = sorted(raw_pred, key=lambda r: -r["score"])
    if raw_pred:
        return raw_pred[0]["label"]
    return None
def predict(examples, predictor, batch_size=128):
    raw_preds, preds, distribution = [], [], []
    with torch.no_grad():
        for e in (range(0, len(examples), batch_size)):
            raw_preds.extend(predictor(examples[e:e+batch_size]))
    for raw_pred in raw_preds:
        raw_pred = raw_pred if type(raw_pred) == list else [raw_pred]
        for m in raw_pred:
            m["label"] = int(m["label"].split("_")[1])
    return raw_preds

p = predict([orig], predictor=pipe)[0]
(p, extract_predict_label(p))

**my output:** 
([{'label': 0, 'score': 0.9998107552528381},
  {'label': 1, 'score': 0.0001892388245323673}],
 _0)_
